explainers/subresource-loading.md

```diff
@@ -78,24 +79,25 @@ give us a better chance of designing the right API.
 This feature is a powerful feature that can replace any subresources in the
 page. So we limit the use of this feature only in [secure contexts](https://www.w3.org/TR/powerful-features/).
 
-## `<link>`-based API
+## `<script>`-based API
```

Comment:
Maybe note somewhere that `<script type="webbundle">` must be inline and the `src=` attribute cannot be used (for now?).

Reply:
A good point! Done.